doc/tracing.md

```diff
@@ -7,21 +7,22 @@ The plugin request address (req) is used to associate requests between NCCL call
 The NCCL request address (request) is used to associate calls from NCCL to the plugin.
 
 To trace aws-ofi-nccl using LTTNG Tracing:
+
 1. Install the LTTNG libraries and dependencies as documented at https://lttng.org/docs/v2.13/#doc-building-from-source
-2. Configure aws-ofi-nccl as normal, but with the addition of the --with-lttng=<prefix> flag to ./configure.  Use as
+1. Configure aws-ofi-nccl as normal, but with the addition of the --with-lttng=<prefix> flag to ./configure.  Use as
```

Comment:
Something went wrong here.

Reply:
This is [intentional](https://mdformat.readthedocs.io/en/stable/users/style.html#ordered-lists) on the part of mdformat to make it possible to insert an element into a list without necessarily changing all preceding elements. All markdown renderers will automatically do the right thing (eg: see the rendered version of the file [here](https://github.com/aws/aws-ofi-nccl/blob/3a5a9846ff1267edd32cca8067c38630358006ee/doc/tracing.md)